lms/static/sass/views/_shoppingcart.scss

```diff
@@ -303,6 +309,34 @@ $light-border: 1px solid $gray-l5;
         color: $light-gray2;
       }
 
+      .course-registration-title,
+      .course-dates-title {
+        @extend %t-title6;
+        display: block;
+        padding: 0;
+        text-transform: uppercase;
+        color: $light-gray2;
+      }
+
+      .course-display-name,
+      .course-display-dates {
+        @extend %t-title4;
+        display: block;
+        color: $dark-gray2;
+      }
+
+      .course-title-info {
```

Comment:
Can you move these layout bits above the styles for the inner bits (insert at line 311)?